README.md

```diff
@@ -21,12 +21,19 @@ Bare minimum features needed to satisfy use case:
 ### Phase 1: Building a House
 *Setup a server that:*
   - *Serves a basic html file with a WebGL canvas saying "hello" in 3D*
-  - *Exposes a basic GraphQL endpoint returning "world"*
-- [ ] Todo items for this phase, in order
+  - *Exposes a basic GraphQL endpoint returning "hello"*
+- [ ] Create a DJango web server
+- [ ] Initialize a GraphQL endpoint & return some nonsense
+- [ ] Write a thin SDK on the frontend to execute API calls
+- [ ] Successfully incorporate nonsense from API in the rendering
+- [ ] Setup a database solution & store some nonsense in it
+- [ ] Return that nonsense through the API
 
 ### Phase 2: Going Grocery Shopping
 *Collect the necessary data from free resources that would be needed for the MVP (for example, what stars can we see from where)*
-- [ ] Todo items for this phase, in order
+- [ ] Store star catalog information in database
+- [ ] Store bortle location data in database
+- [ ] Store elevation data in database / get from somewhere
```

Comment:
unclear what is goal for surrounding elevation (detailed surroundings like street view?), and how to get data (e.g. would be difficult to separate sky from street view api image), unsure about feasibility and necessity for MVP

Reply:
There are elevation scan maps, it would not be an image. Kind of like when it shows you where the buildings and trees are on apple maps (not Satellite mode)

But also I think it would be good to start with just natural elevation data, which is more readily available. This would indicate views blocked by mountains and hills. I remember when I lived in Fremont there was always a mountain in the way of something, and if I didn't live on the hill, the hill would block the other direction..